.github/workflows/evals-nightly.yml

```
@@ -0,0 +1,63 @@
+permissions:
+  contents: read
+
+jobs:
+  worksheet-canary:
+    name: Worksheet charts/maps canary
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    timeout-minutes: 45
+    env:
+      OPENROUTER_API_KEY: ${{ secrets.OPENROUTER_API_KEY }}
+      PROMPTFOO_CONFIG_DIR: ${{ runner.temp }}/promptfoo
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)